presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -238,3 +238,9 @@ Unicode Functions
     are replaced with `replace`. The replacement string `replace` must either
     be a single character or empty (in which case invalid characters are
     removed).
+
+.. function:: key_sampling_percent(varchar) -> double
```

Comment:
Do you also want to introduce a version of this function for `varbinary`?

Reply:
Hmm interesting. I will open an issue for later. For now are using only varchar version.